cvat-core/src/session.ts

```diff
@@ -1939,819 +1946,811 @@ export class Task extends Session {
     }
 }
 
-const {
-    getAnnotations,
-    putAnnotations,
-    saveAnnotations,
-    hasUnsavedChanges,
-    searchAnnotations,
-    searchEmptyFrame,
-    mergeAnnotations,
-    splitAnnotations,
-    groupAnnotations,
-    clearAnnotations,
-    selectObject,
-    annotationsStatistics,
-    importCollection,
-    exportCollection,
-    importDataset,
-    exportDataset,
-    undoActions,
-    redoActions,
-    freezeHistory,
-    clearActions,
-    getActions,
-    closeSession,
-    getHistory,
-} = require('./annotations');
-
 buildDuplicatedAPI(Job.prototype);
 buildDuplicatedAPI(Task.prototype);
 
-Job.prototype.save.implementation = async function () {
-    if (this.id) {
-        const jobData = this._updateTrigger.getUpdated(this);
-        if (jobData.assignee) {
-            jobData.assignee = jobData.assignee.id;
+(async () => {
```

Comment:
Why do we need such changes? I see `Property '...' does not exist on type 'Task'/'Job'` ts error anyway.